editoast/src/views/timetable/stdcm/request.rs

```diff
@@ -112,7 +113,7 @@ pub(crate) struct Request {
     /// Maximum speed of the consist in km/h
     #[serde(default, with = "units::meter_per_second::option")]
     pub(crate) max_speed: Option<quantities::Velocity>,
-    pub(crate) loading_gauge_type: Option<LoadingGaugeType>,
+    pub(crate) tracks: Option<HashSet<String>>,
```

Comment:
Let's rename `tracks` to `zone`. We can also add some documentation to this field using standard Rust documentation `///`.
(We can precise that a `None` value means no zone restriction).